modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/manager/DefaultPortalKaleoManager.java

```diff
@@ -321,7 +324,8 @@ private String _getLocalizedTitle(long companyId, String definitionName)
 		return _localization.getXml(localizedValuesMap, "title");
 	}
 
-	private static final String _DEFINITION_NAME = "Single Approver";
+	private static final String _DEFINITION_NAME =
+		WorkflowDefinitionConstants.NAME_SINGLE_APPROVER;
```

Comment:
Please, remove this variable (`_DEFINITION_NAME`) and use `WorkflowDefinitionConstants.NAME_SINGLE_APPROVER` directly